crates/iceberg-catalog/src/catalog/mod.rs

```diff
@@ -62,6 +61,14 @@ fn require_warehouse_id(prefix: Option<Prefix>) -> Result<WarehouseIdent> {
         .try_into()
 }
 
+pub(crate) fn set_not_found_status_code(
```

Comment:
maybe move this to API? it's working with http stuff

Reply:
Done